client/rest-high-level/src/main/java/org/elasticsearch/client/ml/job/process/TimingStats.java

```diff
@@ -49,12 +50,28 @@
         new ConstructingObjectParser<>(
             "timing_stats",
             true,
-            args ->
-                new TimingStats((String) args[0], (long) args[1], (Double) args[2], (Double) args[3], (Double) args[4], (Double) args[5]));
+            args -> {
+                String jobId = (String) args[0];
+                Long bucketCount = (Long) args[1];
+                Double totalBucketProcessingTimeMs = (Double) args[2];
+                Double minBucketProcessingTimeMs = (Double) args[3];
+                Double maxBucketProcessingTimeMs = (Double) args[4];
+                Double avgBucketProcessingTimeMs = (Double) args[5];
+                Double exponentialAvgBucketProcessingTimeMs = (Double) args[6];
+                return new TimingStats(
+                    jobId,
+                    getOrDefault(bucketCount, 0L),
+                    getOrDefault(totalBucketProcessingTimeMs, 0.0),
+                    minBucketProcessingTimeMs,
```

Comment:
My thinking was that we should provide non-null values wherever possible to simplify dealing with the resulting stats.
We can do it for bucketCount and totalBucketProcessingTimeMs because "0" is natural default value for them.
However, Min, Max and AVG need at least one data point to be calculated so there is no natural default value here.
WDYT?